site/content/docs/developing/observability.en.md

```diff
@@ -0,0 +1,57 @@
+# Observability
+
+## Getting Started
+Copilot can deploy the AWS OpenTelemetry Collector as a [sidecar](./sidecars.en.md) by setting the following in your manifest:
+```yaml
+observability:
+  tracing: awsxray
+```
+
+Copilot will then deploy the AWS OpenTelemetry Collector using the [ECS X-Ray configuration](https://github.com/aws-observability/aws-otel-collector/blob/main/config/ecs/ecs-xray.yaml), which does the following:
+
+- Receives [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) telemetry data on default ports `:4317` (GRPC) and `:4318` (HTTP)
+- Adds [ECS](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#amazon-ecs) and [EC2](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor#aws-ec2) resource information to traces generated by your service
+- [Batches](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) traces before export
+- [Exports](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsxrayexporter) traces to AWS X-Ray
+
+## Instrumenting Your Service
+Instrumenting your service to send telemetry data is done through [language specific SDKs](https://opentelemetry.io/docs/instrumentation/). 
```

Comment:
What do you think of including a JS snippet here from the website or your own code? just so people get a sense of what to look for in the docs

Reply:
yeah i've been going back and forth on that - i'll condense it as much as possible and include a snippet :)